.github/ISSUE_TEMPLATE/story.yaml

```diff
@@ -0,0 +1,45 @@
+name: Story (Internal Use Only 🔒)
```

Comment:
Stories and tasks are often created through the little circle icon in the owning epic or story, so I wonder whether this template can be integrated into that mechanism?

Reply:
Not sure! I can check, but is this required info for the purposes of this code review?

Reply:
Not required, just food for thought.